scripts/deployment/deploy_liquidity_hub.sh

```diff
@@ -36,8 +36,9 @@ function store_artifact_on_chain() {
 	local version=$(cat ''"$contract_path"'/Cargo.toml' | awk -F= '/^version/ { print $2 }')
 	local version="${version//\"/}"
 
-	local res=$($BINARY tx wasm store $artifact $TXFLAG --from $deployer)
-	local code_id=$(echo $res | jq -r '.logs[0].events[] | select(.type == "store_code").attributes[] | select(.key == "code_id").value')
+	local res=$($BINARY tx wasm store $artifact $TXFLAG --from $deployer | jq -r '.txhash')
+	sleep $tx_delay
+	local code_id=$($BINARY q tx $res --node $RPC -o json | jq -r '.logs[0].events[] | select(.type == "store_code").attributes[] | select(.key == "code_id").value')
```

Comment:
now since we use the flag sync, we can't just read directly from res but rather query the tx and parse from there.